default/scripting/species/common/population.macros

```diff
@@ -225,6 +226,19 @@ GASEOUS_BONUS
             effects = SetTargetPopulation value = Value - 0.5*abs(Value)
 '''
 
+MINDLESS_BONUS
+'''     EffectsGroup
+            scope = Source
+            activation = And [
+                Planet
+                Not Planet type = Asteroids
+                HasTag name = "MINDLESS_SPECIES"
+            ]
+            accountinglabel = "MINDLESS_LABEL"
+            priority = [[TARGET_POPULATION_SCALING_PRIORITY]]
+            effects = SetTargetPopulation value = Value - 0.5*abs(Value)
```

Comment:
why not `Value * 0.5`?

Reply:
So that they don't get easier colonization of planets with population malus (e.g. from specials).
Maybe irrelevant, but it makes sense that if they are at -2 they don't make it -1 from their -50% pop.

Reply:
I don't see how it makes any difference, since -1 colony is practically as dead as -2 one.

Reply:
It makes the difference of making sense. Bad population is expected to subtract, not to add, even when the final state is still uncolonizable.
That's the way it is done for good or bad population traits, and it is staying as is.

Reply:
Depending when the scaling priority happen, other stuff could be added later, presumably as well...